UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSEndToEndScenarioFullAvcsPollingTimeOut.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSEndToEndScenarioFullAvcsPollingTimeOut")]
+    public class POSEndToEndScenarioFullAvcsPollingTimeOut
+    {
+        private string fssJwtToken;
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
+        private static readonly FSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
+
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            AuthTokenProvider authTokenProvider = new();
+            fssJwtToken = await authTokenProvider.GetFssToken();
+
+            HttpResponseMessage apiResponse = MockHelper.ConfigureFM(posWebJob.MockApiBaseUrl, posWebJob.FMConfigurationFullAvcsPollingTimeOut);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+            await CommonHelper.RunWebJob();
```

Comment:
this has the 70sec static delay. it would have been better to re engineer to check in a loop on a more frequent basis. That said we said that this will slide for this

Reply:
70 seconds of delay is added to wait so that webjob gets executed before we run the FTs. If we dont add this delay, FTs would get executed before webjob complets and creates batches in stub and that would fail the assertions wrtitten in the FTs.

Generally, webjob takes 50 seconds to get complete for 10 productidentifiers but we have added 20 seconds of buffer time.

Happy to discuss if needed.